.gitignore

```diff
@@ -1,6 +1,8 @@
 .buildlog
+.dart_tool/
 .DS_Store
```

Comment:
Points for cleaning this up, too

.DS_Store, .idea, .settings, .buildlog – these can all get removed